### PR TITLE
[12.0][MIG] account_move_line_purchase_info

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Migration of module account_move_line_purchase_info to v12

### CODE REVIEW

**Review by tbaden — CHANGES_REQUESTED** [review]
https://github.com/OCA/account-financial-tools/blob/29ef7116d8505baa0d3a8ca89679b7a64848c702/account_move_line_purchase_info/models/__init__.py#L1-L5

please sort these alphabetically (switch move and invoice). anything else looks fine so far.

**Comment by AdriaGForgeFlow** [comment]
> please sort these alphabetically (switch move and invoice). anything else looks fine so far.

Thanks for the review :+1: but I would like to avoid making aesthetics changes because once you open that door there's no coming back.

**Review by tbaden — APPROVED** [review]
it's in the contribution guidelines, yes it really is a minor "issue".
final call is with the oca. approving for now.

**Review by amcor — APPROVED** [review]
Code review only

File: account_move_line_purchase_info/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#           (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Move Line Purchase Info",
+    "summary": "Introduces the purchase order line to the journal items",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.github.com/OCA/account-financial-tools",
```
Comment (nikul-serpentcs) [inline account_move_line_purchase_info/__manifest__.py]: `https://github.com/OCA/account-financial-tools`

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM! :+1:

**Pull request merged**